app/design/frontend/Elogic/elogic_theme/Magento_Cms/web/css/source/_module.less

```diff
@@ -97,47 +138,78 @@
             }
         }
         &__text {
-            color: @color-drexler-gray;
-            font-size: @font-size_s;
-            letter-spacing: .75px;
             padding-bottom: 50px;
+        }
+        &__link {
             
         }
+    }
+     //Categories============================================================
+
+     .categories{
+
+         &__container {
+        }
+        &__body {
+            .lib-vendor-prefix-display(flex);
+        }
+    
+    }
+
+    .item-category {
+        background-color: @color-drexler-lightGray;
+        height: 312px;
+        margin: 15px;
+        .lib-vendor-prefix-display(flex);
+        justify-content: space-around;
+
+        &__content {
+            .lib-vendor-prefix-display(flex);
+            .lib-vendor-prefix-flex-direction (column);
+            align-items: flex-start;
+            padding-top: 100px;
+            padding-right: 20px;
+        }
+
+        &__img {
+            padding-right: 20px;
+            padding-left: 30px;
+            img {
+                max-width: 300px;
+            }
+           
+        }
+        &__title {
+            margin-bottom: 10px;
```

Comment:
for margins/paddings check these variables:
@indent__xs
@indent__s
@indent__base
@indent__l
@indent__xl